doc/manuals/admin/statistics.md

```diff
@@ -0,0 +1,215 @@
+If a particular request type has not been received, its corresponding counter is not shown.
+
+### SemWait block
+
+It provides accumulated waiting time for main internal semaphores. It can be useful to detect bottelnecks, e.g.
+if your DB is two slow or your DB pool is undersized, then `dbConnectionPool` time would be anormaly high
+(see section on [performance tunning](perf_tuning.md) for more details).
+
+```
+{
+  ...
+  "semWait" : {
+    "request" : "0.000000000",
+    "dbConnectionPool" : "2.917002794",
+    "transaction" : "0.567478849",
+    "subCache" : "0.784979145",
+    "curlContext" : "0.000000000",
+    "timeStat" : "0.124000605"
+  },
+  ...
+}
+```
+
+### Timing block
+
+Provides timing information, i.e. the time that CB passes executing in different internal modules.
+
+```
+{
+  ...
+  "timing": {
+    "accumulated": {
+      "jsonV1Parse": "7.860908311",
+      "mongoBackend": "416.796091597",
+      "mongoReadWait": "4656.924425628",
+      "mongoWriteWait": "259.347915990",
+      "mongoCommandWait": "0.514811318",
+      "render": "108.162782114",
+      "request": "186476.593504743",
+      "xmlParse": "44.490766332"
+        },
+    "last": {
+      "mongoBackend": "0.014752309",
+      "mongoReadWait": "0.012018445",
+      "mongoWriteWait": "0.000574611",
+      "render": "0.000019136",
+      "request": "0.015148915",
+      "xmlParse": "0.000153878"
+      }
+  }
+  ...
+}
+```
+
+The block includes two main sections:
+
+* `last`: times corresponding to the last processed request. If that request didn't used a particular module
+  (e.g. a GET request doesn't use parsing), then that counter is 0 and it is not shown.
+* `accumulated`: accumulated time corresponding to all requests.
+
+The particular counters are as follow:
+
+* `request`: processing time for the whole request, excluding the time that the HTTP library
+  takes for request/response dispatching (pseudo end-to-end time)
+* `jsonV1Parse`: time passed in NGSIv1 JSON parsing module (pseudo self-time)
+* `jsonV2Parse`: time passed in NGSIv2 JSON parsing module (pseudo self-time)
+* `xmlParser`: time passed in NGSIv1 XML module (pseudo self-time)
+* `mongoBackend`: time passed in mongoBackend module (pseduo self-time)
+* `render`: time passed in rendering module (pseudo self-time)
+* `mongo*Wait`: time passed waiting for MongoDB for `Read`, `Write` or `Cmd` operations. Note that if
+  a given request involves several read/write/cmd calls to MongoDB, the time shown in `mongo*Wait` under
+  `last` includes the accumulation for all them.
+
+Times are measured since a particular thread request start using a module and ends using it. Thus, if the
+thread is stoped for any reason (e.g. the kernel decides to give priority to another thread based on its
+scheculing policy) the time is not accurate. That is what we say *pseudo* selt/end-to-end time. However,
+under low load conditions this situation is not expected to have a significative impact.
+
+### NotifQueue block
```

Comment:
@crbrox  please have a look specially to this section, in order to check if the different counter are correctly described.